backends/hif/pk-backend-hif.c

```diff
@@ -186,7 +187,7 @@ pk_backend_setup_hif_context (HifContext *context, GKeyFile *conf, const gchar *
 	hif_context_set_cache_dir (context, cache_dir);
 	solv_dir = g_build_filename (destdir, "/var/cache/PackageKit", release_ver, "hawkey", NULL);
 	hif_context_set_solv_dir (context, solv_dir);
-	repo_dir = g_build_filename (destdir, "/etc/yum.repos.d", NULL);
+	repo_dir = g_build_filename (destdir, hif_get_reposdir(), NULL);
```

Comment:
That's not the point of this code or pull request. `hif_get_reposdir()` returns only a string. For Fedora, it returns `/etc/yum.repos.d`. For Mageia, it returns `/etc/distro.repos.d`. If the mechanism of retrieving reposdirs changes in PackageKit-Hif, then this function should change as well.